IntegrationDevelopment/Help/IntegrationDevelopmentGuide.postman_collection.json

```diff
@@ -5788,6 +5788,434 @@
 						}
 					],
 					"description": "This folder contains examples from the [Organization Management (REST API Example)](https://help.acumatica.com/Help?ScreenId=ShowWiki&pageid=afa51ca5-37ae-4d7e-aa36-761779ca9320) chapter of the Integration Development Guide."
+				},
+				{
+					"name": "Manufacturing",
+					"item": [
+						{
+							"name": "Creation of Bills of Material: Sign In",
```

Comment:
Why you have separate sign in and sign out requests for manufacturing? They should not differ from the corresponding requests for other endpoints.